Description: Correct some values of Plasmoid metadata
Author: Peter J. Mello <admin@petermello.net>
Origin: other
Forwarded: no
Last-Update: 2020-01-08

--- plasma-applet-qalculate-0.8.0.orig/plasmoid/metadata.desktop
+++ plasma-applet-qalculate-0.8.0/plasmoid/metadata.desktop
@@ -2,17 +2,18 @@
 Encoding=UTF-8
 Name=Qalculate!
-Comment=Qalculate! applet for plasma desktop
+Comment=Qalculate! applet for Plasma desktop
+Icon=qalculate
 Type=Service
+
 X-KDE-ParentApp=
-X-KDE-PluginInfo-Author=DSchopf
+X-KDE-PluginInfo-Author=Daniel Schopf
+X-KDE-PluginInfo-Category=Utilities
 X-KDE-PluginInfo-Email=schopf.dan@gmail.com
-X-KDE-PluginInfo-License=GPL
+X-KDE-PluginInfo-License=MIT
 X-KDE-PluginInfo-Name=org.kde.plasma.qalculate
-X-KDE-PluginInfo-Version=0.1
-X-KDE-PluginInfo-Website=plasma.kde.org
+X-KDE-PluginInfo-Version=0.8.0
+X-KDE-PluginInfo-Website=https://store.kde.org/p/1155946
 X-KDE-ServiceTypes=Plasma/Applet
+
 X-Plasma-API=declarativeappletscript
 X-Plasma-MainScript=ui/Qalculate.qml
-X-Plasma-RemoteLocation=
-X-KDE-PluginInfo-Category=Utilities
-Icon=/usr/share/plasma/plasmoids/org.kde.plasma.qalculate/contents/images/Qalculate.svg
